config/htmlmin.php

```diff
@@ -46,4 +46,25 @@
 
     'force' => false,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Ignore Blade Files
+    |--------------------------------------------------------------------------
+    |
+    | Here you can specify paths, which you don't want to minimize, for example
+    | e-mails or markdowns.
+    |
+    */
+
+    'ignore' => [
+        'resources/views/emails',
+        'resources/views/html',
+        'resources/views/notifications',
+        'resources/views/markdown',
+        'resources/views/vendor/emails',
+        'resources/views/vendor/html',
+        'resources/views/vendor/notifications',
+        'resources/views/vendor/markdown',
```

Comment:
@GrahamCampbell Well, if it cannot be in this package this way, I can just remove all 8 hard-coded paths from the config, and everyone can set it up themselves. But keeping these would make it more plug-and-play, as people wouldn't have to investigate these folder names themselves (so it's clean code vs usability question). How do you prefer?